src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,88 @@
-# TO-DO: Complete the selection_sort() function below 
-def selection_sort( arr ):
-    # loop through n-1 elements
-    for i in range(0, len(arr) - 1):
-        cur_index = i
-        smallest_index = cur_index
-        # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
+import copy
+# TO-DO: Complete the selection_sort() function below
 
 
+def selection_sort(arr):
+    for i in range(0, len(arr) - 1):
+        start = arr[i]
+        # print(arr)
+        for j in range(i + 1, len(arr)):
+            if arr[i] > arr[j]:
+                arr[i], arr[j] = arr[j], arr[i]
 
```

Comment:
This one looks like an implementation of a bubble sort to me.

That is, your inner for-loop is continuously 'bubbling' the larger item towards the end of the array/list.

Instead for selection, your inner for loop should be finding what the smallest value is and then right after the loop would be swapping that smallest value found with the value at index i.